lisa_commands/commands.py

```diff
@@ -501,14 +526,54 @@ def shame(query, slack_event):
         {'user': {'S': query}}
     )['Item']['karma']['N']
 
-def spotify_me(query, _slack_event):
+def spotify_me(query, _slack_event, query_type=None):
     """
     query - query str
     slack_event - A dict of slack event information(unused for this function)
 
```

Comment:
Like the approach add a doc for query-type here. (give us the name, type, what it is, and why it's optional)

Reply:
Right, docs, duh. Updated